cmd/podman/containers/container.go

```diff
@@ -20,7 +17,7 @@ var (
 		RunE:             registry.SubCommandExists,
 	}
 
-	defaultContainerConfig = getDefaultContainerConfig()
+	defaultContainerConfig = util.DefaultContainerConfig()
```

Comment:
```suggestion
	containerConfig = util.DefaultContainerConfig()
```
In other packages, we have containerConfig as a PodmanConfig which carries a config.Config().  I suspect we dont' want `util.DefaultContainerConfig() `here rather `registry.PodmanConfig()`.